src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -252,13 +251,13 @@ func runClientProxy(cfg *config.Config) {
 	}()
 
 	/*
-		Temporarily disabling localdiscover. See:
-		https://github.com/getlantern/lantern/issues/2813
-		// Continually search for local Lantern instances and update the UI
-		go func() {
+		      Temporarily disabling localdiscover. See:
+		      https://github.com/getlantern/lantern/issues/2813
+		      // Continually search for local Lantern instances and update the UI
+		      go func() {
 			addExitFunc(localdiscovery.Stop)
 			localdiscovery.Start(!showui, strconv.Itoa(tcpAddr.Port))
-		}()
+		      }()
 	*/
```

Comment:
Looks like this was here before, but maybe we just remove this actually? IMO, it's rarely helpful keeping commented out code around as its just creates unnecessary clutter.